packages/arch/packages/input/src/Input.js

```diff
@@ -35,13 +35,14 @@ export const inputStyles = (props: InputProps = {}) => ({
   ...(props.isMultiline
     ? {
         lineHeight: 'inherit',
-        height: 'auto',
+        height: 100,
```

Comment:
Limit to vertical resize:
```suggestion
        height: 100,
        resize: 'vertical',
```